src/xhp/Custombranding.php

```diff
@@ -0,0 +1,18 @@
+<?hh // strict
+
+class :custombranding extends :x:element {
+  category %flow;
+
+  protected string $tagName = 'custombranding';
+
+  protected function render(): XHPRoot {
+    $custom_text = \HH\Asio\join(Configuration::gen('custom_text'));
+    $custom_image = \HH\Asio\join(Configuration::gen('custom_logo_image'));
```

Comment:
I think it'd make more sense to move this out of this XHP element, and pass it in through an attribute. In general, we shouldn't be introducing more `join`s into the code, only `await`.